testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/xerces/ws/unit/XercesUsageInWebServiceTestCase.java

```diff
@@ -71,6 +73,17 @@ public static WebArchive createWebServiceDeployment() {
         war.addClasses(XercesUsageWebService.class, XercesUsageWSEndpoint.class);
         // add a web.xml containing the webservice mapping as a servlet
         war.addAsWebInfResource(XercesUsageServlet.class.getPackage(), "xerces-webservice-web.xml", "web.xml");
+        // add parmisions
+        war.addAsManifestResource(
+                PermissionUtils.createPermissionsXmlAsset(UndertowServiceActivator.appendPermissions(
```

Comment:
Why does a deployment need all these permissions?  These don't seem the kind thing a deployment should need to be aware of.

Reply:
For some reason IBM jdk fail for me. Id assume because deployment include xerces?